### PR TITLE
PRO-2253 pr template

### PR DESCRIPTION
**Pull request by abea**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!-- Please don't delete this template -->
+
+<!-- PULL REQUEST TEMPLATE -->
+<!-- (Update "[ ]" to "[x]" to check a box) -->
+
+## Summary
+
+*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Close #[issue number]"*
```
Comment (gregvanbrug) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should this be:
> ... "Closes #[issue number]"*

I think [either works](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) but "Closes" sounds more grammatically correct.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
+> 4. Check that all pieces have been archived properly
+
+## What kind of change does this PR introduce?
+*(Check at least one)*
+
+- [ ] Bugfix
+- [ ] Feature
+- [ ] Code style update
+- [ ] Refactor
```
Comment (gregvanbrug) [inline .github/PULL_REQUEST_TEMPLATE.md:23]: Code style update and Refactor kinda feel the same. What is the difference between these two changes?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+- [ ] Docs
+- [ ] Build-related changes
+- [ ] Other, please describe:
+
+## Make sure the PR fulfills these requirements:
+
+- [ ] When resolving a specific issue, it references the issue in the PR's title (e.g. `resolves #xxx[,#xxx]` for Github issues, `PRO-xxx` for internal Linear tickets -- where "xxx" is the issue number)
+
+**If there is no related issue/ticket,** the PR's summary includes:
```
Comment (gregvanbrug) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should this be part of the next item so the list could be uninterrupted?
```
- [ ] **If there is no related issue/ticket,** the PR's summary includes a convincing reason for adding this feature or clear description of the bug it resolves
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+- [ ] A convincing reason for adding this feature or clear description of the bug it resolves
+
+- [ ] The Changelog is updated
+- [ ] Related documentation has been updated
+- [ ] Related tests have been updated
+
+To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.
```
Comment (gregvanbrug) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should this read something more like:

```
If adding a new feature, it's best to open a **feature request issue** first and wait for approval before working on it.
```

Alright to leave as is but the way it is currently written has a bit of negative connotation to me.

**Review by gregvanbrug — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (abea) [reply .github/PULL_REQUEST_TEMPLATE.md:23]: No huge difference necessarily. I see "refactor" as possibly being more complex than code style, which might be a ton of spacing fixes and var=>const for eslint. I'll remove one to keep things simpler.

**Review by gregvanbrug — APPROVED** [review]

**Pull request merged**